cd/application-deployment/dev/dev.env

```diff
@@ -43,6 +44,7 @@ TEMPLATE_SERVICE_PROVIDERS_ENABLED=True
 TWILIO_ACCOUNT_SID=fake
 TWILIO_AUTH_TOKEN=fake
 UI_HOST_NAME=https://dev.notifications.va.gov
+V3_ENABLED=True
```

Comment:
I noticed that all the other environments explicitly set this to False, which is the default.  I'm including it here for consistency and so at least one environment allows us to test V3.